### ISSUE TITLE
Experience for using VC

### ISSUE THREAD

**Issue by Galakey** [Nekorooni/Kiara#37]
25exp/minute of being in VC, so 1500exp/hour.

**Comment by Nekorooni** [comment]
There's no real way to track actual voice activity right now, so people can possible just afk in the voice channels for free xp

**Comment by Sharpienero** [comment]
@Nekorooni Make the timeout shorter than the pay period. i.e make the pay period time = 30 minutes, but have timeout after 15 minutes of inactivity.

**Comment by Nekorooni** [comment]
@Sharpienero I'm assuming you mean the afk timeout. iirc that's not based on voice activity either but away status, you can listen to some music bot while muted and it doesn't move you.

**Comment by Sharpienero** [comment]
@Nekorooni From what I saw when doing research a bit ago, it goes off the "voice" indicator bubble. i.e time since last activity.

I wonder if it ignores the user if the "mute microphone" button is toggled.

**Comment by Nekorooni** [comment]
But the thing is, anyone could just afk in a channel listening to music or
something, it won't make a difference if they actually talk or not. They're
not guaranteed to be even listening at all too.

**Comment by Sharpienero** [comment]
@Nekorooni You're right, there is no completely reliable and accurate way to track it.
